common-lib/src/cmr/common/api/errors.clj

```diff
@@ -66,33 +64,46 @@
   [error]
   (update-in error [:path] keyword-path->string-path))
 
-(defmethod errors->body-string mt/json
-  [_ errors]
-  (json/generate-string {:errors (map error->json-element errors)}))
+;; *************************************
 
 (defmulti error->xml-element
   "Converts an individual error element to the equivalent XML structure."
   type)
 
 (defmethod error->xml-element String
   [error]
-  (x/element :error {} (mask-token-error error)))
+  (xml/element :error {} (mask-token-error error)))
```

Comment:
Can you just convert the three functions into 1 and use a case statement? It will execute much faster that way.  defmulti/defmethods are not needed here.

Reply:
I converted the last one, the other two are based on java classes and I did not have much luck with that.